### PR TITLE
First draft of IPFS Weekly #98

### PR DESCRIPTION
**Pull request by jennwrites**
Please give tips/suggestions/feedback by 12pm Central on Monday, July 27!

To be distributed Tuesday, July 28 around 12pm Central.

### CODE REVIEW

**Review by serejandmyself — COMMENTED** [review]
## People build the coolest stuff with IPFS
* A decentralized, hardcore, uncensored [Twitter analogue](https://cyber.page/ipfs/QmdfKnMkQUDgBa5rK7UiJG4xULRAKttUsEcDJkq5trotjm) that stores your content in IPFS, built on top of the [Cyber protocol](https://cyber.page)

File: published/098-2020-jul-28.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+[Project Lead, libp2p](https://jobs.lever.co/protocol/27ff3891-6e13-4aa8-b43a-734715e85a26): This is a unique opportunity to take ownership of a massively successful open source project and carry it through its next phase of growth. Protocol Labs, Remote
+
+[Engineering Manager, IPFS](https://jobs.lever.co/protocol/3f0787e8-58b3-4122-a1ea-424561d2658f): Engineering managers have the challenging and exciting task of supporting and empowering engineering teams to deliver complex internet-scale systems in an environment defined by curiosity, passion, and a love for open source. Protocol Labs, Remote
+
+[IPFS Community Lead](https://jobs.lever.co/protocol/71c4a9b9-af90-4ce9-9dba-8b72507997bf): Provide tactical support for the IPFS community, creating a global community strategy, and executing this strategy on the ground! Protocol Labs, Remote
+
+[Senior Software Engineer Rust/Go](https://www.notion.so/Hiring-Senior-Software-Engineer-Rust-Go-e6c94ccc261f426c80a483c7fc642412): You would be working with our distributed team of engineers on new products, core protocols or client projects. Most of the work we do is or will eventually be open source. Equilibrium, Remote
+
```
Comment (carsonfarmer) [inline published/098-2020-jul-28.md:57]:
```suggestion

[Signed and Encrypted IPLD, Go](https://www.notion.so/Signed-and-Encrypted-data-in-IPFS-e1593e90b56e44c38e165109999782ce): Implement a new Go-based IPLD codec that supports signed and encrypted Blocks, and get paid for it! Externally funded, 3Box and Textile, Remote
```

**Review by carsonfarmer — CHANGES_REQUESTED** [review]
This seems like a clean way to add a suggestion @renrutnnej? Let me know if you'd prefer a proper PR.

**Comment by jennwrites** [comment]
Thanks @serejandmyself and @carsonfarmer! ☺️

**Pull request merged**